R/data.R

```diff
@@ -168,13 +177,18 @@ spim_lancelot_data_rtm <- function(date, region, model_type, data, full_data) {
 
   if (region %in% c("northern_ireland", "scotland", "wales", "uk")) {
     data$deaths <- data$death2
+    data[, deaths_hosp_age] <- NA_integer_
     data$deaths_hosp <- NA_integer_
     data$deaths_comm <- NA_integer_
     data$deaths_carehomes <- NA_integer_
     data$deaths_non_hosp <- NA_integer_
   } else {
     data$deaths <- NA_integer_
-    data$deaths_hosp <- data$death3
+    if (all(!is.na(data[, deaths_hosp_age])) && !full_data) {
```

Comment:
I think this would be better off moved to the `if(!full_data)` section after `ret <- data.frame(...`